docs/mobile-apps/features/fast-booted-simulators.md

```diff
@@ -0,0 +1,110 @@
+
+1. On Sauce Labs click **Live** > **Cross Browser** > **Mobile Virtual**.
+   
+<img src={useBaseUrl('img/mobile-apps/fast-sim-1.png')} alt="fast booted menu" width="750"/>
+
+2. Select “Apple” as the Manufacturer.
+3. If you want to use the “current_major” version then use these values :
+- Select **iPhone 14 Simulator** for the Device.
+- Select the latest available version of iOS.
+
+
+:::note 
+If you want to use the “previous_major” version then use these values:
+- Select “iPhone 13 Simulator” for the Device
+- Select “iOS 15.4” for the OS Version
+:::
+
+## Accessing Fast Booted iOS Simulator for Automated Testing with Appium
+
+To use the fast-booted simulators, specify the following device configuration capabilities:
+
+For iPhone:
+
+```java
+"appium:deviceName": "iPhone Fast Simulator"
+"appium:platformVersion": "latest|current_major|previous_major"
+```
+
+For iPad:
+
+```java
+"appium:deviceName": "iPad Fast Simulator"
+"appium:platformVersion": "latest|current_major|previous_major"
+```
+
+This will assign a fast-booted Simulator that represents the latest/current major iOS version for the requested device by what both Apple has released and what Sauce Labs supports in our Simulator cloud. In the case of using previous_major platform version, it will represent the latest previous major iOS version for the requested device by what both Apple has released and what Sauce Labs supports in our Simulator cloud.
+
+:::note
+To use a different version of Appium, you can specify it through the [sauce:options](https://docs.saucelabs.com/dev/test-configuration-options/#appiumversion). Check our [platform configurator](https://saucelabs.com/products/platform-configurator#/) to confirm which Appium versions are supported. Note that using a different version may result in additional 20 seconds of start-up time for your iPhone/Ipad Fast Simulator.
+:::
+
+
+## FAQ
+
+
+### How can I validate that my session used a Fast Booted Simulator?
+
+This can be done by:
+
+- Validating the session command time, which should be between 0-10 seconds. The video should start with a booted sim.
+  
+<img src={useBaseUrl('img/mobile-apps/fast-sim-2.png')} alt="fast booted menu" width="750"/>
+
+
+
+- Opening the Metadata tab, as Sauce Labs automatically add tags.
+If you are using `“appium:deviceName”: “iPhone Fast Simulator”` or `“appium:deviceName”: “iPad Fast Simulator”`, you should see the tag **fast-simulator**. When you use the default Appium version, you will additionally see the **fast-simulator-with-matching-appium** tag.
+<img src={useBaseUrl('img/mobile-apps/fast-sim-3.png')} alt="fast booted menu" width="750"/>
+
+### I set up a Fast Booted Simulator, but it’s still taking a long time to boot.
+
+
+There are a few reasons why this could happen. The first reason is that all available Fast Booted Simulators are already used and your session uses the default way of starting a Simulator. 
```

Comment:
```suggestion
There are a few reasons why this could happen. The first reason is that all available Fast Booted Simulators are already used and your session uses the default way of starting a Simulator.
```